test/engine/test_execute.py

```diff
@@ -391,9 +391,10 @@ def test_not_an_executable(self):
         for obj in (
             Table("foo", MetaData(), Column("x", Integer)),
             Column("x", Integer),
-            tsa.and_(),
+            # a single clause in and_ is optimized away
```

Comment:
**Federico Caselli** (CaselIT) wrote:

Won't this short circuit the and_ and test something else?

Reply:
**mike bayer** (zzzeek) wrote:

ah you're correct in this case, because and_(True) produces an AsBoolean object and I missed that.   But I think what's being tested here is not really specific to ClauseList, it's just testing that a variety of random ClauseElement things are not going to try to invoke a statement.  this is kind of an old test, it's not that critical.

Reply:
**Federico Caselli** (CaselIT) wrote:

Done